python/cucim/src/cucim/skimage/util/dtype.py

```diff
@@ -279,7 +279,7 @@ def _convert(image, dtype, force_copy=False, uniform=False):
     #   is a subclass of that type (e.g. `cp.floating` will allow
     #   `float32` and `float64` arrays through)
 
-    if cp.issubdtype(dtype_in, cp.obj2sctype(dtype)):
+    if cp.issubdtype(dtype_in, np.core.numerictypes.obj2sctype(dtype)):
```

Comment:
Does this still handle CuPy types smoothly? Does NumPy version matter here?

Reply:
[Yes, because cp.obj2sctype is just an alias for the NumPy function](https://github.com/cupy/cupy/blob/5d5ef648d1b1b3af4c969e2b7a47b0d63f8b57b0/cupy/__init__.py#L363-L380). The same is true for various dtypes like `cp.float32`, etc.